specs/elements/BaseElement.md

```diff
@@ -11,9 +11,19 @@ Abstract class for the base element type.
 
 # Rendering
 
-1. If any required elements are missing or invalid, engage fallback and stop.
-1. Else, render the element
-1. If failed to render, engage fallback and stop.
+1. If the element is going to render without any contents, renderers MUST **drop** the element, SHOULD trigger a warning, and MUST NOT trigger fallback.
+	1. What counts for "not rendering any contents"?
+		1. Container that has `spacing` and `separator` but nothing else --> **DROP**
+		1. Container that has `minHeight` but nothing else --> **Keep**. Author might be using it for explicit spacing?
+		1. Image who's image fails to load --> **DROP**
```

Comment:
Compared to today...

* JS: CHANGE, today it collapses to 0 height but still reserves space around it, not truly dropped
* UWP: Seemingly gets dropped but that's because of bug #3254
* .NET: CHANGE, today it collapses to 0 height but still reserves space around it, not truly dropped